### PR TITLE
Preventing C#10 in VS 2022 as <LangVersion> defaults to latest

### PR DESCRIPTION
**Pull request by LucGenetier**

### CODE REVIEW

File: src/libraries/Microsoft.PowerFx.Connectors/Microsoft.PowerFx.Connectors.csproj [hunk 1]
```diff
@@ -10,6 +10,7 @@
   <!-- Nuget Properties -->
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>9.0</LangVersion>
```
Comment (lesaltzm) [inline src/libraries/Microsoft.PowerFx.Connectors/Microsoft.PowerFx.Connectors.csproj:13]: This is a common property, can we just put it in the top level Directory.Build.props?

Comment (LucGenetier) [reply src/libraries/Microsoft.PowerFx.Connectors/Microsoft.PowerFx.Connectors.csproj]: Done

**Comment by BruceHaley** [comment]
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.Core.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.Interpreter.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.LanguageServerProtocol.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.Transport.Attributes.dll**

File: src/libraries/Microsoft.PowerFx.Transport.Attributes/Microsoft.PowerFx.Transport.Attributes.csproj [hunk 2]
```diff
@@ -33,7 +33,7 @@
   <!-- Nuget Properties -->
   <PropertyGroup>
     <PackageId>Microsoft.PowerFx.Transport.Attributes</PackageId>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```
Comment (lesaltzm) [inline src/libraries/Microsoft.PowerFx.Transport.Attributes/Microsoft.PowerFx.Transport.Attributes.csproj:36]: Nit: looks like a bit of extra white space was added in all these csproj files. 
run git checkout main — ‘*.csproj’ to clean that up

**Comment by BruceHaley** [comment]
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.Core.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.Interpreter.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.LanguageServerProtocol.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.Transport.Attributes.dll**

**Review by lesaltzm — APPROVED** [review]

**Comment by BruceHaley** [comment]
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.Core.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.Interpreter.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.LanguageServerProtocol.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.PowerFx.Transport.Attributes.dll**

**Pull request merged**